lib/1.4/dml-builtins.dml

```diff
@@ -1391,6 +1391,20 @@ a private helper object.
 The `init_as_subobj` template accepts one parameter,
 `classname`. This defines the class of the automatically created object.
 
+<div class="note">
+
+**Note:** The subobject class defined by the `classname` parameter is looked up
+using `SIM_get_class` while the module of the device class is being
+loaded.
+This may cause problems if the subobject class is defined within the
+same module as the device class:
+If the device class is defined before the subobject class, then the subobject
```

Comment:
nit: `if`, not `If`.

Reply:
My feeling was that both are valid and I didn't know which one is better. ChatGPT chose your side so I changed